HRSPLT-237 Tweak leave balance tab text about Earnings Statement.

apetro opened the pull request:
It turns out "Earnings Statement" is a preferred term over "pay check".

apetro committed:
HRSPLT-237 Tweak leave balance tab text about Earnings Statement.

timlevett commented:
This text is supposed to inform them when this data was last refreshed (as of the last pay period). The text now says the balances are available in there stubs. Now they don't know when the data below was refreshed forcing them to use the other module. Seems a bit odd.

apetro committed:
*also* available.

Implies the currency relationship of these balances to those on the
latest earnings statement.

timlevett commented:
status update?

timlevett commented:
Taking #50 instead
